.github/workflows/docs.yml

```diff
@@ -0,0 +1,21 @@
+on: [push, pull_request]
```

Comment:
You probably want to enable these tests only on `push`s to `master`. Otherwise this will run whenever any branch is pushed to `kornia/kornia`. This applies to all workflows in this PR.

Reply:
not sure if that happens that often

Reply:
As of writing this comment, you have ~15 branches in this repository besides `master`. I can't comment on how often they are used or updated.

Reply:
this is a good reminder to make some house cleaning :)